src/rapaio/math/linear/dense/EigenvalueDecomposition.java

```diff
@@ -42,7 +41,7 @@
  * RV.times(D). The matrix RV may be badly conditioned, or even singular, so the
  * validity of the equation A = RV*D*inverse(RV) depends upon RV.cond().
  */
-@Deprecated
```

Comment:
I see no thoroughful testng of this decomposition. Unless it is properly tested, this code should not be touched. Otherwise it would be close to impossible to find a bug in a refactored code. I think this decomposition should remain the same